### PR TITLE
Summary

### PR DESCRIPTION
**Pull request by Xenans**

### CODE REVIEW

File: frontend/src/components/App.js [hunk 1]
```diff
@@ -39,17 +39,19 @@ class App extends Component {
         super(props);
 
         this.state = {
-            items: [],
-            users: []
+            items: [{id:1, name:"420", price:"420", users:[1]}, {id:2, name:"69", price:"69", users:[2]}],
```
Comment (tnc11) [inline frontend/src/components/App.js:42]: please remove test data

**Review by tnc11 — CHANGES_REQUESTED** [review]
everything looks good, but please remove test data.

**Review by tnc11 — APPROVED** [review]
thanks for removing the test data. Looks good to me

**Comment by Xenans** [comment]
np lets get this merge

**Pull request merged**